ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt

```diff
@@ -0,0 +1,148 @@
+
+1227.0 "Thermal Suppression" sync / 1[56]:[^:]*:Lugae:62FA:/
+1232.6 "Magitek Ray" sync / 1[56]:[^:]*:Lugae:62FC:/
+1244.1 "Magitek Explosive" sync / 1[56]:[^:]*:Lugae:62F8:/
+1255.1 "Explosion" sync / 1[56]:[^:]*:Magitek Explosive:62F9:/
+
+1266.9 "Downpour" sync / 1[56]:[^:]*:Lugae:62F5:/ window 66.9,10
+1274.4 "Magitek Missile" sync / 1[56]:[^:]*:Lugae:62F6:/
+1278.7 "Surface Missile" sync / 1[56]:[^:]*:Lugae:62F7:/
+1279.9 "Magitek Missile" sync / 1[56]:[^:]*:Lugae:62F6:/
+1284.2 "Surface Missile" sync / 1[56]:[^:]*:Lugae:62F7:/
+1285.4 "Magitek Missile" sync / 1[56]:[^:]*:Lugae:62F6:/
+1289.7 "Surface Missile" sync / 1[56]:[^:]*:Lugae:62F7:/
+1294.9 "Thermal Suppression" sync / 1[56]:[^:]*:Lugae:62FA:/ window 30,30
+1300.4 "Magitek Ray" sync / 1[56]:[^:]*:Lugae:62FC:/
+1312.0 "Magitek Explosive" sync / 1[56]:[^:]*:Lugae:62F8:/
+1323.0 "Explosion" sync / 1[56]:[^:]*:Magitek Explosive:62F9:/
+
+1335.3 "Magitek Chakram" sync / 1[56]:[^:]*:Lugae:62F3:/ jump 1200.0
+1343.8 "Magitek Chakram"
+1346.3 "Mighty Blow"
+1354.9 "Mighty Blow"
+1362.4 "Thermal Suppression"
+1368.0 "Magitek Ray"
+
+#~~~~~~~#
+# ANIMA #
+#~~~~~~~#
+
+# -ii 52BE 5E68 6305
+
+2000.0 "--sync--" sync / 00:0839::Iron Womb will be sealed off/ window 2000,1
+2009.8 "Lunar Nail" sync / 1[56]:[^:]*:Anima:62FE:/ 2009.8,10
+2020.7 "Phantom Pain" sync / 1[56]:[^:]*:Anima:62FF:/
+2029.8 "Mega Graviton" sync / 1[56]:[^:]*:Anima:6300:/
+2037.8 "Pater Patriae" sync / 1[56]:[^:]*:Anima:6306:/
+2043.7 "Aetherial Pull" sync / 1[56]:[^:]*:Mega-graviton:6301:/ window 2043.7,10
+2045.8 "Graviton Spark" sync / 1[56]:[^:]*:Mega-graviton:6302:/
+2058.4 "Boundless Pain" sync / 1[56]:[^:]*:Anima:6303:/
+2058.5 "--sync--" sync / 1[56]:[^:]*:Anima:6675:/
+2059.6 "Boundless Pain" sync / 1[56]:[^:]*:Anima:6304:/ duration 11
+2082.5 "Imperatum" sync / 1[56]:[^:]*:Anima:6309:/ window 2082.5,10
+2085.0 "--sync--" sync / 1[56]:[^:]*:Anima:5D79:/
+2109.8 "Obliviating Claw #1" sync / 1[56]:[^:]*:Anima:630B:/
+2116.6 "Obliviating Claw #2" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2121.6 "Obliviating Claw #3" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2127.6 "Charnel Claw #1" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2132.6 "Charnel Claw #2" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2135.1 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/
+2143.6 "Obliviating Claw" sync / 1[56]:[^:]*:Anima:630A:/
+2149.1 "Coffin Scratch x5" sync / 1[56]:[^:]*:Anima:630E:/ duration 5
+2155.7 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/
+2164.2 "Obliviating Claw #1" sync / 1[56]:[^:]*:Anima:630B:/
+2171.0 "Obliviating Claw #2" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2176.0 "Obliviating Claw #" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2182.0 "Charnel Claw #1" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2187.0 "Charnel Claw #2?" sync / 1[56]:[^:]*:Iron Nail:630D:/ # can be skipped?
+2189.5 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/ window 20,20
+2199.1 "Oblivion" sync / 1[56]:[^:]*:Anima:630F:/ duration 5
+2205.8 "--sync--" sync / 1[56]:[^:]*:Anima:5D40:/ window 2205.8,30
```

Comment:
Is this an hp% based phase? If it is, can you add a TODO that there may be more mechanics?

Can you also add a brief comment or whitespace here to separate out the different phases?